static/js/components/CreatePostForm_test.js

```diff
@@ -1,11 +1,10 @@
-// @flow
```

Comment:
Just curious, what did flow complain about here?

Reply:
in a lot of places in the codebase we declare the props for our components like this:

```js
class Foo extends React.Component<*, void> {
  props: FooProps

 ...
}
```

it turns out that, when you type a component like that, it type-checks the props for methods *inside* the component but doesn't type check the props being passed in. So obviously this is not at all what we want. Instead, what we want to do is this:

```js
class Foo extends React.Component<FooProps, void> {
  ...
}
```

I noticed this when I was working on the detail page changes, and I've been changing it as I go through things since then. But basically, once I made that it change it started complaining about props that we missing in the test file and so on, so I decided to just turn off flow for the test file